packages/remix-server-runtime/routeModules.ts

```diff
@@ -13,9 +13,9 @@ export interface RouteModules<RouteModule> {
 /**
  * The arguments passed to ActionFunction and LoaderFunction.
  */
-export interface DataFunctionArgs {
+export interface DataFunctionArgs<Context = unknown> {
```

Comment:
```suggestion
export interface DataFunctionArgs<Context = any> {
```

While `unknown` is better, it would be a breaking change. Let's change this to `any` for the time being.